backend/coins/btc/handlers/handlers.go

```diff
@@ -580,3 +583,102 @@ func (handlers *Handlers) postSetTxNote(r *http.Request) (interface{}, error) {
 
 	return nil, handlers.account.SetTxNote(args.InternalTxID, args.Note)
 }
+
+type signingResponse struct {
+	Success      bool   `json:"success"`
+	Signature    string `json:"signature"`
+	Aborted      bool   `json:"aborted"`
+	ErrorMessage string `json:"errorMessage"`
+	ErrorCode    string `json:"errorCode"`
+}
+
+func (handlers *Handlers) postEthSignMsg(r *http.Request) (interface{}, error) {
+	var signInput string
+	if err := json.NewDecoder(r.Body).Decode(&signInput); err != nil {
+		return signingResponse{Success: false, ErrorMessage: errp.WithStack(err).Error()}, nil
```

Comment:
the stack is actually not printed like this nor should it. `err.Error()` should be the same